src/utils/date-utils.js

```diff
@@ -7,6 +7,7 @@ import advancedFormat from 'dayjs/plugin/advancedFormat'
 const KNOWN_FORMATS = {
   onlyDate: 'DD/MM/YYYY',
   iso: 'YYYY-MM-DD',
+  custom: 'DD MMMM HH:mm',
```

Comment:
hmm 4 M? which format is that?

Reply:
Ah it's the month in letters. https://github.com/iamkun/dayjs/blob/dev/docs/en/API-reference.md#list-of-all-available-formats